doc/source/io.rst

```diff
@@ -79,116 +79,140 @@ for some advanced strategies
+  - ``keep_date_col``: boolean, default ``False``. If ``True`` and parse_dates
+    specifies combining multiple columns then keep the original columns.
+  - ``date_parser``: function, default ``None``. Function to use for converting
+    a sequence of string columns to an array of datetime instances. The default
+    uses dateutil.parser.parser to do the conversion. Pandas will try to call
+    date_parser in three different ways, advancing to the next if an exception
+    occurs: 1) Pass one or more arrays (as defined by parse_dates) as arguments;
+    2) concatenate (row-wise) the string values from the columns defined by
+    parse_dates into a single array and pass that; and 3) call date_parser once
+    for each row using one or more strings (corresponding to the columns defined
+    by parse_dates) as arguments.
+  - ``dayfirst``: boolean, default ``False``. DD/MM format dates, international
+    and European format.
+  - ``thousands``: str, default ``None``. Thousands separator.
+  - ``comment``: str, default ``None``. Indicates remainder of line should not
+    be parsed. If found at the beginning of a line, the line will be ignored
+    altogether. This parameter must be a single character. Like empty lines (as
+    long as ``skip_blank_lines=True``), fully commented lines are ignored by the
+    parameter `header` but not by `skiprows`. For example, if ``comment='#'``,
+    parsing '#empty\\na,b,c\\n1,2,3' with `header=0` will result in 'a,b,c'
+    being treated as the header.
+  - ``decimal``: str, default ``'.'``. Character to recognize as decimal point.
+    E.g. use ``','`` for European data.
+  - ``nrows``: int, default ``None``. Number of rows of file to read. Useful for
+    reading pieces of large files.
+  - ``iterator``: boolean, default ``False``. Return `TextFileReader` object
+    for iteration or getting chunks with ``get_chunk()``.
+  - ``chunksize``: int, default ``None``. Return `TextFileReader` object for
+    iteration. See :ref:`iterating and chunking <io.chunking>` below.
+  - ``skipfooter`` : int, default ``0``. Number of lines at bottom of file to
+    skip (unsupported with engine='c').
+  - ``converters``: dict, default ``None``. Dict of functions for converting
+    values in certain columns. Keys can either be integers or column labels.
+  - ``verbose``: boolean, default ``False``. Indicate number of NA values placed
+    in non-numeric columns.
+  - ``delimiter``: str, default ``None``. Alternative argument name for sep.
+    Regular expressions are accepted.
+  - ``encoding``: str, default ``None``. Encoding to use for UTF when
+    reading/writing (e.g. ``'utf-8'``). `List of Python standard encodings
+    <https://docs.python.org/3/library/codecs.html#standard-encodings>`_.
+  - ``squeeze``: boolean, default ``False``. If the parsed data only contains
+    one column then return a Series.
+  - ``na_filter``: boolean, default ``True``. Detect missing value markers
```

Comment:
let's put the NA options (``na_values``,``na_filter``,``keep_default_na``) near each other (change the code as well to match this ordering)